builtin/mainmenu/keyringmgr.lua

```diff
@@ -0,0 +1,206 @@
+keyringmgr = {
+  keyring = nil,
+
+	general_login = false
```

Comment:
Please fix the indentation of this file (should be tabs).

See also https://docs.luanti.org/for-engine-devs/lua-code-style-guidelines/.